conda/moose/run_test.sh

```diff
@@ -1,7 +1,47 @@
 #!/bin/bash
-set -e
-combined-opt --copy-inputs tests
-cd combined/tests
+
+# If Intel Mac, forget about it. The environment doesn't seem fit to run
+if [[ $(uname) == 'Darwin' ]] && [[ $(uname -m) == 'x86_64' ]]; then
+    exit 0
+fi
+
+# A list of module(s)/directories that we do not want to include for testing
+NOT_RUNNABLE=(doc module_load combined geochemistry misc navier_stokes tensor_mechanics framework external_petsc_solver reactor_tutorial)
```

Comment:
this should be captured in an issue if it has not already

Reply:
https://github.com/idaholab/moose/issues/16857